pset-functions.js

```diff
@@ -136,8 +164,40 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const word = "function"
+const scoreKey = {a: 1, b: 3, c: 3, d: 2, e: 1, f: 4, g: 2, h: 4, i: 1, j: 8, k: 5, l: 1, m: 3, n: 1, o: 1, p: 3, q: 10, r: 1, s: 1, t: 1, u: 1, v: 4, w: 4, x: 8, y: 4, z: 10}
+
+function scrabbleScore(word) {
+
+
+const wordArray = word.split ('')
```

Comment:
@Ashbk32 be careful with your indentation, it's off here...code that lives inside of other code (i..e inside the opening and closing  curly braces) should be indented